packages/repack/ios/ScriptManager.mm

```diff
@@ -2,23 +2,26 @@
 #import "ScriptConfig.h"
 #import "ErrorCodes.h"
 
+// make it work with use_frameworks!
```

Comment:
Maybe it's worth to create a GH issue and leave a todo?

Reply:
I've just added an explicit comment here, the fix was done with different PR :)